plugin/helpers_bots.go

```diff
@@ -64,3 +65,43 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 
 	return createdBot.UserId, nil
 }
+
+func (p *HelpersImpl) KVGetJSON(key string, value interface{}) error {
+	data, err := p.API.KVGet(key)
+	if err != nil {
+		return err
+	}
+
+	return json.Unmarshal(data, value)
+}
+
+func (p *HelpersImpl) KVSetJSON(key string, value interface{}) error {
+	data, err := json.Marshal(value)
+	if err != nil {
+		return err
+	}
+	return p.API.KVSet(key, data)
+}
+
+func (p *HelpersImpl) KVCompareAndSetJSON(key string, oldValue interface{}, newValue interface{}) error {
```

Comment:
I think we want to return the boolean from `KVCompareAndSet` here, otherwise the caller won't know if the assignment was successful.